lbz/dev/server.py

```diff
@@ -5,22 +5,27 @@
 import json
 import logging
 import urllib.parse
+from abc import ABCMeta, abstractmethod
 from http.server import BaseHTTPRequestHandler
 from http.server import HTTPServer
-from typing import Tuple, Union
+from typing import Tuple, Union, Type
 
 from lbz.dev.misc import Event
 from lbz.resource import Resource
 from lbz.response import Response
 
 
-class MyLambdaDevHandler(BaseHTTPRequestHandler):
+class MyLambdaDevHandler(BaseHTTPRequestHandler, metaclass=ABCMeta):
     """
     Mimics AWS Lambda behavior.
     """
 
-    cls = None
-    done = False
+    done: bool = False
+
+    @property
+    @abstractmethod
+    def cls(self) -> Type[Resource]:
```

Comment:
WTF

Reply:
to enforce definition when inhariteded instead of leaving it as none.

Reply:
Probably, I understand your point of view but I think this pattern makes more sense for _"real"_ abstract classes, not for enforcing one thing in fully implemented ones. I have mixed feelings about this 🤔

Reply:
There is no suggestion on how to proceed ?

Reply:
The suggestion would be to not do things like this at all, at least not in the pull request which is responsible for adding types 🙂  We can improve this part of the code (including the naming convention) in the near future if really needed.

Reply:
[comment #685879018](https://github.com/pdyba/lambdalizator/pull/37#discussion_r685879018)
> The suggestion would be to not do things like this at all, at least not in the pull request which is responsible for adding types 🙂 We can improve this part of the code (including the naming convention) in the near future if really needed.

_Bumped, just to not forget about that._

Reply:
actually - it makes sense to add here as then it would be even more problematic for mypy
as we would have to use Union or Optional 
cls: Type(Resource) = None - wont allow us to progress.

Reply:
You can always do: `cls: Type[Resource] = None  # type: ignore` 😉  and it would be the easiest thing in this case.

But ok, let's say, we will improve it in the future, just to simplify the code.